internal/pkg/cli/pipeline_init.go

```diff
@@ -118,11 +117,15 @@ func (opts *InitPipelineOpts) Validate() error {
 // Execute writes the pipeline manifest file.
 func (opts *InitPipelineOpts) Execute() error {
 	secretName := opts.createSecretName()
-	secretArn, err := opts.secretsmanager.CreateSecret(secretName, opts.GitHubAccessToken)
+	_, err := opts.secretsmanager.CreateSecret(secretName, opts.GitHubAccessToken)
```

Comment:
Do we not need to capture the secret ARN on successful creates any more?

Reply:
we never used it -- see https://github.com/aws/amazon-ecs-cli-v2/pull/390#discussion_r347111774